shorebird_code_push/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: shorebird_code_push
 description: Check for and download Shorebird code push updates from your app.
-version: 0.1.3
+version: 1.0.0
```

Comment:
I suspect we're going to make lots more breaking changes to the APIs soon, but I don't really care what the version number is.

Reply:
Yeah, this is just a) being Correct™ with semver and b) removing doubt that it's ready for use